.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -39,12 +39,12 @@ stages:
         - bash: |
             SKU_NAME=${OS_VERSION} && \
             if [[ "${HYPERV_GENERATION,,}" == "v2" ]]; then SKU_NAME="${SKU_NAME}gen2"; fi && \
-            if [[ ${OS_VERSION} == "V2" && ${ARCHITECTURE,,} == "arm64" && ${OS_SKU} == "CBLMariner" ]]; then SKU_NAME="${SKU_NAME}arm64"; fi && \
```

Comment:
I wonder why the ossku conditional was there in the first place?

Reply:
my guess is that it's there to strengthen the condition check to really ensure that this condition is triggered for Marinerv2 ARM64. Since AzureLinux is now added into the picture, decided to remove it since V2 is only specific to Mariner/AzureLinux and no need to further complicate the logic here.